KWZP2022/KWZP2022/FormResDepParametryCzesc.cs

```diff
@@ -0,0 +1,161 @@
+            cmbCzesc.ValueMember = "ID_czesc";
+            cmbParametr.DataSource = db.v_Rodzaj_parametr.ToList();
+            cmbParametr.DisplayMember = "Parametr";
+            cmbParametr.ValueMember = "ID_rodzaj_parametr";
+        }
+        private void initDataGridView()
+        {
+            dgvParametrCzesc.DataSource = db.v_Parametry_czesc.ToList();
+            dgvParametrCzesc.Columns[0].Visible = false;
+            dgvParametrCzesc.Columns[1].Visible = false;
+            dgvParametrCzesc.AutoResizeColumns(DataGridViewAutoSizeColumnsMode.AllCells);
+            this.dgvParametrCzesc.AutoSizeColumnsMode = System.Windows.Forms.DataGridViewAutoSizeColumnsMode.Fill;
+        }
+
+        private void btnDodaj_Click(object sender, EventArgs e)
+        {
+            try
+            {
+                if (txtZakresGora.Text == null || txtZakresDol.Text == null || Convert.ToDecimal(txtZakresGora.Text) < Convert.ToDecimal(txtZakresDol.Text))
+                {
+                    MessageBox.Show("Wpisz poprawny zakres", "Błąd", MessageBoxButtons.OK);
+                }
+                else
+                {
+                    Parametr_czesc parametrCzesc = new Parametr_czesc();
+                    parametrCzesc.ID_czesc = (int)cmbCzesc.SelectedValue;
+                    parametrCzesc.ID_rodzaj_parametr = (int)cmbParametr.SelectedValue;
+                    parametrCzesc.Zakres_gora = Convert.ToDecimal(txtZakresGora.Text);
+                    parametrCzesc.Zakres_dol = Convert.ToDecimal(txtZakresDol.Text);
+                    db.Parametr_czesc.Add(parametrCzesc);
+                    db.SaveChanges();
+                    initDataGridView();
+                }
+            }
+            catch
+            {
+                MessageBox.Show("Powtórzono parametr", "Błąd", MessageBoxButtons.OK);
+            }
+        }
+
+        private void btnWczytajParametr_Click(object sender, EventArgs e)
+        {
+            string czesc = cmbCzesc.SelectedValue.ToString();
+            int czescID = int.Parse(czesc);
+            System.Linq.IQueryable vParametrCzesc = db.v_Parametry_czesc.Where(x => x.ID_czesc == czescID);
+            int vParametrCzescInt = vParametrCzesc.Cast<v_Parametry_czesc>().Where(x => x.ID_czesc > 0).Count();
+            if (vParametrCzescInt > 0)
+            {
+                dgvParametrCzesc.DataSource = vParametrCzesc.Cast<v_Parametry_czesc>().ToList();
+            }
+        }
+
+        private void btnUsun_Click(object sender, EventArgs e)
+        {
+            DialogResult dialogResult = MessageBox.Show("Czy na pewno chcesz usunąć parametr: " + this.dgvParametrCzesc.CurrentRow.Cells[3].Value + "?", "Question", MessageBoxButtons.YesNo);
+            if (dialogResult == DialogResult.Yes)
+            {
+                string current_parametr = this.dgvParametrCzesc.CurrentRow.Cells[1].Value.ToString();
+                int daneINT = int.Parse(current_parametr);
```

Comment:
wszystko tutaj to `dane`, proszę o nazwę zmiennej

Reply:
zmieniono